flang/unittests/Lower/OpenMPLoweringTest.cpp

```diff
@@ -41,4 +41,19 @@ TEST_F(OpenMPLoweringTest, Barrier) {
   EXPECT_EQ(succeeded(barrierOp.verify()), true);
 }
 
+TEST_F(OpenMPLoweringTest, TaskWait) {
```

Comment:
Is it possible to share some code between these tests?

Reply:
It can be done(by clustering directives in one test and check) but IMO that will defeat the purpose of unit test. right ?
The common code that can be shared across all is already in fixture.